Data/CommonContext.cs

```diff
@@ -0,0 +1,16 @@
+using BizzDirectory.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace BizzDirectory.Data
+{
+    public class CommonContext : DbContext
+    {
+        public CommonContext(DbContextOptions<CommonContext> options) : base(options) { }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<Organization>().ToTable("Organization");
```

Comment:
what will it do?

Reply:
it creates a table "organization" matching with the "Organization" class